projects/kit/directives/data-list-dropdown-manager/data-list-dropdown-manager.directive.ts

```diff
@@ -26,13 +26,12 @@ import {
     tap,
 } from 'rxjs/operators';
 
-// TODO: 3.0 Move into separate module
 @Directive({
     selector: `tui-data-list[tuiDataListDropdownManager]`,
 })
 export class TuiDataListDropdownManagerDirective implements AfterViewInit {
     @ContentChildren(TuiDropdownDirective, {descendants: true})
-    private readonly dropdowns: QueryList<TuiDropdownDirective> = EMPTY_QUERY;
+    private readonly dropdowns: QueryList<any> = EMPTY_QUERY;
```

Comment:
"any" is not a mistake?

Reply:
Yeah, I'm in the middle of refactoring it and it was `any` in my branch to avoid type issues :) fixed